File: KW_20_03_24/V1.js
const readline = require('readline-sync');
const s = readline.question("(1, 2,").split(",");
const a = parseInt(s[0]);
const b = parseInt(s[1]);
const c = parseInt(s[2]);

const numbers = [a, b, c];
if (a === b && b === c) {
    console.log("Числа равны");
} else {
    const max_num = Math.max(...numbers);
    const min_num = Math.min(...numbers);
    let avg_num;
    if (a !== max_num && a !== min_num) {
        avg_num = a;
    } else if (b !== max_num && b !== min_num) {
        avg_num = b;
    } else {
        avg_num = c;
    }
    const equal_count = numbers.filter(num => num === a).length - 1 + numbers.filter(num => num === b).length - 1 + numbers.filter(num => num === c).length - 1;
    console.log(`Максимум - ${max_num}, Минимум - ${min_num}, Среднее - ${equal_count > 0 ? 'Его нет' : avg_num}, Одинаковых - ${equal_count}`);
}

File: KW_20_03_24/V1.py
s = input().split(",")
a = int(s[0])
b = int(s[1])
c = int(s[2])

numbers = [a, b, c]
if a == b == c:
    print("Числа равны")
else:
    max_num = max(numbers)
    min_num = min(numbers)
    if a != max_num and a != min_num:
        avg_num = a
    elif b != max_num and b != min_num:
        avg_num = b
    else:
        avg_num = c
    equal_count = numbers.count(a) - 1 + numbers.count(b) - 1 + numbers.count(c) - 1
    print(f"Максимум - {max_num}, Минимум - {min_num}, Среднее - {'Его нет' if equal_count > 0 else avg_num}, Одинаковых - {equal_count}")
